recipes/soci/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class SociConan(ConanFile):
+    name = "soci"
+    homepage = "https://github.com/SOCI/soci"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "The C++ Database Access Library "
+    topics = ("C++", "database-library", "oracle", "postgresql", "mysql", "odbc", "db2", "firebird", "sqlite3", "boost" )
+    license = "Boost"
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    options = {
+        "fPIC":       [True, False],
+        "static":     [True, False],
+        "cxx11":      [True, False],
+        "empty":      [True, False],
+        "shared":     [True, False],
+        "sqlite3":    [True, False],
+        "db2":        [True, False],
+        "odbc":       [True, False],
+        "oracle":     [True, False],
+        "firebird":   [True, False],
+        "mysql":      [True, False],
+        "postgresql": [True, False],
+        "boost":      [True, False],
```

Comment:
```suggestion
        "with_sqlite3":    [True, False],
        "with_db2":        [True, False],
        "with_odbc":       [True, False],
        "with_oracle":     [True, False],
        "with_firebird":   [True, False],
        "with_mysql":      [True, False],
        "with_postgresql": [True, False],
        "with_boost":      [True, False],
```

It's a pattern used for most recipes.